test/e2e/helpers.js

```diff
@@ -531,17 +531,7 @@ const PRIVATE_KEY_TWO =
 const ACCOUNT_1 = '0x5cfe73b6021e818b776b421b1c4db2474086a7e1';
 const ACCOUNT_2 = '0x09781764c08de8ca82e156bbf156a3ca217c7950';
 
-const defaultGanacheOptions = {
-  accounts: [
-    {
-      secretKey: PRIVATE_KEY,
-      balance: convertETHToHexGwei(DEFAULT_GANACHE_ETH_BALANCE_DEC),
```

Comment:
this is '25ETH' to the account corresponding to the default SRP (both ganache and anvil), so it can be removed, as it's in the node class level, and adding this to a spec has no effect